File: cells-with-odd-values-in-a-matrix.js
/**
 * @param {number} m
 * @param {number} n
 * @param {number[][]} indices
 * @return {number}
 */
var oddCells = function(m, n, indices) {
  let mat = Array(m).fill(Array(n).fill(0))

  indices.forEach(([a, b]) => {
    console.log(m)
    for (let i = 0; i < m; i++) {
      console.log(mat)
      mat[i][b]+=1
      console.log(mat)
    }



  })
  let ans = 0

  mat.forEach((line, i) => line.forEach((el, j) => el % 2 === 1 && ans++))

  return ans
}

File: go-archive/cells-with-odd-values-in-a-matrix.go
func oddCells(m int, n int, indices [][]int) int {
    ans:=0; matrix:=[][]int{}

    for i:= 0; i < m; i++ {
        line:=[]int{}
        for j:=0; j < n; j++ {
            line = append(line, 0)
        }
        matrix = append(matrix, line)
    }

    for i:=0; i < len(indices); i++ {
        a:= indices[i][0]; b:= indices[i][1]

        for j:=0; j < n; j++ {
            matrix[a][j]++
        }

        for j:=0; j < m; j++ {
            matrix[j][b]++
        }
    }

    for i:= 0; i < m; i++ {
        for j:=0; j < n; j++ {
            if (matrix[i][j] % 2 == 1) {ans++}
        }
    }

    return ans
}